CONTRIBUTING.md

```diff
@@ -89,6 +89,21 @@ Contributors may provide database identifiers associated with each member of the
 
 Several files are described as being classic [biom-format](http://www.biom-format.org) tables. In these files, the first line must begin with the text ``#Taxonomy``, followed by a tab-separated list of one or more sample identifiers. All sample identifiers provided here must be present in ``sample-metadata.tsv``. Each subsequent line should begin with the taxonomic name, followed by a tab-separated list of the relative abundances in each sample. The relative abundances must sum to 1.000 (to three decimal places) for each sample.
 
+## Raw Data
+
+mockrobiota does not host raw data files (e.g., sequencing files). All sequencing data and other raw data files must be deposited on public, external websites. Stable, public depositories are preferred, but this requirement is not enforced by mockrobiota. mockrobiota ensures that valid, accessible links are provided in the dataset metadata (if not, integrity checks will fail and your dataset will not be accepted), but does not manage these external resources and will not guarantee the validity of raw data that are contributed by outside users. When preparing raw data for linking to mockrobiota datasets, please observe the following regulations:
+
+1. All raw sequence data should be deposited in .fastq format and archived using standard compression formats, e.g., .gz or .zip
+2. Mock community datasets that contain multiple samples must be provided in non-demultiplexed files (i.e., one file per read direction per sequencing run, containing multiple uniquely barcoded samples). 
+3. Index/barcode sequences must be provided as a separate .fastq file. If QUAL scores do not exist for these reads, please note this in the human-readable-description field of dataset-metadata.tsv for that dataset.
+4. Reverse sequencing reads are accepted, but not required. Forward and reverse reads should be submitted as separate files, not as joined reads.
+5. All raw data must conform to the following naming conventions: 
+    - mock-forward-read.fastq.gz
+    - mock-reverse-read.fastq.gz
+    - mock-index-read.fastq.gz
+
+In other words, all datasets **should** be associated with three raw data files, following the naming conventions above.
```

Comment:
Drop this line, the above text is really clear so I think this is just confusing.